app/controllers/todos_controller.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class TodosController < ApplicationController
+  rescue_from ActiveRecord::RecordInvalid, with: :record_invalid_handler
```

Comment:
It's nice that we have handling for the default error messages. I suggest here a modular approach. See https://medium.com/rails-ember-beyond/error-handling-in-rails-the-modular-way-9afcddd2fe1b section -> #Error Handling — Modular Approach


But what happens when we have specific cases like validation errors/auth errors/etc. for specific endpoints?